package.json

```diff
@@ -5,6 +5,7 @@
 		"@oclif/command": "^1.5.19",
 		"@oclif/config": "^1.15.1",
 		"@smartthings/core-sdk": "^0.3.0",
+		"cli-table": "^0.3.1",
```

Comment:
Was this done using some form of `lerna add cli-table`? I tried locally, and `lerna` does not seem to add to the root, which leads me to believe it isn't expecting it to work that way and would rather specify in lower packages and hoist them up.

Reply:
I just added it manually and did a `full_clean.sh; bootstrap.sh` I wouldn't expect lerna to add this to the root package.json. Why do you think it should?

Reply:
Weird...I must have put that in accidentally. I've removed it now.